gossip/state/state.go

```diff
@@ -761,6 +761,11 @@ func (s *GossipStateProviderImpl) addPayload(payload *proto.Payload, blockingMod
 	}
 
 	if !blockingMode && payload.SeqNum-height >= uint64(s.config.StateBlockBufferSize) {
+		if s.straggler(height, payload) {
+			s.logger.Warningf("[%s] Detected a distant block sequence (%d), we are lagging behind (%d) and state transfer is disabled in configuration "+
```

Comment:
Getting better. But while you know what a 'block sequence' is in this context, an admin or SRE looking at the log won't. Could you put it into terms that the admin or SRE will understand? E.g. current peer block height.

Reply:
Can you perhaps spell it out for me? What exactly do you want me to write?

Reply:
Something like this:

`[mychannel] Current block height [100] is lagging too far behind other peers at height [500] to be able to get blocks via peer-to-peer gossip. Consider setting the peer to be a leader (peer.gossip.orgLeader = true) so that it receives blocks from the ordering service. Alternatively enable state transfer (peer.gossip.state.enabled = true) so that the peer pulls older blocks from other peers.`

I may not have gotten it exactly correct since I don't know the logic in this part of the code...